### DISCUSSION TITLE
Making sprites visibile in the sprite list when they are doubleclicked in the Stage

### DISCUSSION THREAD

**Discussion by sfederici** [ScratchAddons/ScratchAddons#7828] [Ideas]
In order to access the code/costumes/sounds tabs of a given sprite that is visible on the Stage you can doubleclick it. This operation highlights the sprite thumbnail in the sprite list under the Stage. But, if you are not interested in changing the sprite's scripts/costumes/sounds and you want, e.g., to
- copy scripts to the sprite
- export the whole sprite
when there a lot of sprites in your project (something that happens very often to my students as, in order to simplify their sprite' scripts, they create many duplicates of the same character) you have to slowly scroll the sprite list and look for the highlighted sprite.
This could be made really easier if the highlighed sprite, as soon as you doubleclick it, would be immediately visible in the sprite list by scrolling the list at the right point.

**Comment by DNin01** [comment]
So basically, scroll the sprite card in the sprite pane that is selected (by double-clicking it on the stage) into view.

**Reply by sfederici** [reply]
> So basically, scroll the sprite card in the sprite pane that is selected (by double-clicking it on the stage) into view.

RIght. Exactly
